web/src/components/storage/VolumeForm.jsx

```diff
@@ -70,12 +70,12 @@ const SizeUnitFormSelect = ({ units, ...formSelectProps }) => {
  * @param {Array<import(~/clients/storage).Volume>} props.volumes - a collection of storage volumes
  * @param {object} props.formSelectProps - @see {@link https://www.patternfly.org/components/forms/form-select#props}
  * @returns {ReactComponent}
- */
+*/
 
 const MountPointFormSelect = ({ volumes, ...selectProps }) => {
   return (
     <Select { ...selectProps }>
-      { volumes.map(v => <SelectOption key={v.mountPath} value={v.mountPath}>{v.mountPath}</SelectOption>) }
+      { volumes.map(v => <SelectOption key={v.mountPath} value={v.mountPath} label={v.mountPath} />) }
```

Comment:
There are some issues there I couldn't find more details regarding the select component. I was testing the code on how these changes were affecting the overall code.

Reply:
Ok, I can start helping you by making commits, do you agree?

In this case, I think you need to apply this change:

```suggestion
      { volumes.map(v => <SelectOption key={v.mountPath} value={v.mountPath}>{v.mountPath}</SelectOption>) }
```

Reply:
BTW, how the UI looks at this time for you when adding and editing a file system?

Reply:
Okay sir, Thank you for the help.
I will change it right away.

Reply:
> BTW, how the UI looks at this time for you when adding and editing a file system?

It's showing errors
![Screenshot 2024-01-09 181416](https://github.com/openSUSE/agama/assets/125825782/ee81ca0f-f38d-420b-93b6-6bdb27a96a78)

Reply:
I see, that's because you're missing some important props for the PatternFly/`Select` component like `toggle`. Please, have a look to the previous work done by @joseivanlopez at https://github.com/openSUSE/agama/blob/927352ba83c0d7cc2db6efd3ab693c7a1f140947/web/src/components/storage/VolumeForm.jsx#L223-L230 but, most important, to the [`Select` documentation](https://www.patternfly.org/components/menus/select) for understanding how the component work.

Once you have it more clear we can continue.

Reply:
BTW, since I know you're learning I'd suggest to use more descriptive commit messages. We do not have an official _style guide_ for them in Agama, but in [YaST was suggested](https://yastgithubio.readthedocs.io/en/latest/contributing/#commits) 

> Each commit in the pull request should do only one thing, which is clearly described by its commit message. Especially avoid mixing formatting changes and functional changes into one commit. When writing commit messages, adhere to [widely used conventions](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).

Think that these messages are kind of documentation too. Meaningful commits are an important piece of the version control system.

If you're curious, there are other conventions like https://www.conventionalcommits.org/en/v1.0.0/ which I'm thinking to propose for the project.

Reply:
> BTW, since I know you're learning I'd suggest to use more descriptive commit messages. We do not have an official _style guide_ for them in Agama, but in [YaST was suggested](https://yastgithubio.readthedocs.io/en/latest/contributing/#commits)
> 
> > Each commit in the pull request should do only one thing, which is clearly described by its commit message. Especially avoid mixing formatting changes and functional changes into one commit. When writing commit messages, adhere to [widely used conventions](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
> 
> Think that these messages are kind of documentation too. Meaningful commits are an important piece of the version control system.
> 
> If you're curious, there are other conventions like https://www.conventionalcommits.org/en/v1.0.0/ which I'm thinking to propose for the project.

Okay sir, Thank you for the suggestion.